app/services/getstream_webhook_service.rb

```diff
@@ -0,0 +1,115 @@
+# Parse getstream.io webhook reqest
+class GetstreamWebhookService
+  attr_reader :activities, :activity, :feed_id, :feed_target, :locale, :actor_id
+
+  def initialize(feed)
+    @activities = feed['new']
+    @feed_id = feed['feed'].split(':').last
+    @feed_target = User.find_by(id: @feed_id)
+    @locale = (@feed_target&.language || 'en').to_sym
```

Comment:
Since we won't be expanding locales for the time being, does this fallback to `en` if the locale file isn't found?

If not, we should just force `en`